.github/workflows/ci.yml

```diff
@@ -35,7 +35,22 @@ jobs:
       - test-web-app
       - test-wasm
     runs-on: ubuntu-latest
+    if: always()
     steps:
+      - name: Debug needs
```

Comment:
What's that ?

(don't we risk of leaking sensitive env vars with a bad configuration on this ?)

Reply:
It's a task that print the `needs` context more info [here](https://docs.github.com/en/actions/learn-github-actions/contexts#needs-context).

tl,dr: The needs context contains only the final state a jobs (if it fail or not) it's output (actually none of our jobs are configuring outputs variable)

Reply:
done